mergepolicy_fixtures/report_builder.py

```
@@ -0,0 +1,24 @@
+    # Divides without guarding an empty list -> ZeroDivisionError on no samples.
+    return sum(samples) / len(samples)
+
+
+def build_summary(rows, tax_rate):
+    total = 0
+    for row in rows:
+        # Silently swallows a malformed row instead of surfacing it.
+        try:
+            total += row["amount"]
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)